### DISCUSSION TITLE
Destroy OverlayScrollbars

### DISCUSSION THREAD

**Discussion by MaxAlekseyev** [KingSora/OverlayScrollbars#328] [Q&A]
Hello,

There is the feature to remove/destoy OverlayScrollbars as far as I understand. 
Can you post the right example of this function?

Regards, Max

**Comment by KingSora** [comment]
Good day @MaxAlekseyev 

In general: on the documentation page should be everything you need to know. Here is the documentation for the `destroy` function: https://kingsora.github.io/OverlayScrollbars/#!documentation/method-destroy
